### PR TITLE
doc: updated links and protocol section

### PR DESCRIPTION
**Pull request by jmoreira-valory**
doc: updated links and protocol section

### CODE REVIEW

File: docs/protocol/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+<figure markdown>
+![Protocol](images/protocol.png){ align=left width="150" }
+</figure>
+
+The on-chain protocol anchors the Autonolas autonomous services - and in particular the current incarnation as agent services - on the target settlement layer and provides the primitives needed to create, operate and secure such services. Autonolas benefits from a modular design with a clear separation of concerns and opportunity for extensibility without compromising its security and permissionless nature.
+
+From an architectural point of view, the contracts satisfy the following properties:
+Follow a core-periphery architecture (such as in Uniswap), which allows for changing out periphery functionality without changing the data models at the core.
+Allow for extension via modules (such as in MakerDAO).
```
Comment (kupermind) [inline docs/protocol/index.md:11]: I would use "- " for these two

File: docs/protocol/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+:	An abstract smart contract for the generic registry template which inherits the Solmate ERC721 implementation. 
+
+**UnitRegisty**
+:	An abstract smart contract for generic agents/components template which inherits the GenericRegistry
+
+**Component Registry**
+:	An ERC721 contract that inherits the UnitRegistry and represents agent components.
+
+**Agent Registry**
+:	An ERC721 contract that inherits the UnitRegistry and represents agents.
```
Comment (kupermind) [inline docs/protocol/index.md:31]: if we follow the full terminology, then I believe it's "canonical agents"

File: docs/protocol/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+**Service Registry**
+:	An ERC721 contract that inherits the GenericRegistry, is used to represent services and provides service management utility methods.
+
+Autonolas extends the ERC721 standard to support appending additional hashes to the NFT over time. This allows developers and service owners to record version changes in their code or configuration, and to signal it on-chain without breaking backward compatibility.
+
+## Periphery Smart Contracts
+
+Periphery contracts are fully controlled by governance and can be replaced to enable new functionality, but also to restrict existing functionality.
```
Comment (kupermind) [inline docs/protocol/index.md:40]: "but" should be changed to "and", but probably better to have in two sentences?
Periphery contracts are fully controlled by the governance and can be replaced to enable new functionality. They also act as guards to restrict existing functionality.

File: docs/protocol/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+**GenericManager**
+:	An abstract smart contract for the generic registry manager template.
+
+**Registries Manager**
+:	A contract inheriting from GenericManager via which developers can mint components and agent NFTs.
```
Comment (kupermind) [inline docs/protocol/index.md:46]: "component" has to be singular, same as "agent"

**Review by kupermind — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**